meinberlin/assets/scss/components/_list.scss

```diff
@@ -0,0 +1,10 @@
+.list--highlight {
+    padding-bottom: 2em;
+
+    background-color: $bg-tertiary;
+
+    .filter-bar:first-child {
+        position: relative;
+        top: -0.8em;
+    }
```

Comment:
Good idea, just had to figure out how to not have it always since it's in its own include. 
<img width="754" alt="screen shot 2017-04-04 at 13 05 12" src="https://cloud.githubusercontent.com/assets/16354712/24653854/d2601484-1937-11e7-8efa-4f216ebd48d3.png">
Looks okay to me. I mean it's not too desirable, but then again, I doubt having *that* many filters is desirable either.